skrifa/src/outline/glyf/hint/instance.rs

```diff
@@ -171,13 +171,40 @@ impl HintInstance {
             Definition::default(),
         );
         self.cvt.clear();
+        if let Some(cvar) = outlines.cvar.as_ref() {
```

Comment:
not sure it's useful here, but something that I have found helpful for the idea of "I need both of these optional things to exist" is,

```rust
if let Some((one, two)) = opt1.zip(opt2) {
...
}
```

Reply:
ah, hadn't seen this before. I don't see how I'd use it here but will definitely add it to the toolbox!